apps/src/lib/kits/maker/ui/SetupInstructions.jsx

```diff
@@ -97,6 +114,79 @@ class ConnectionInstructions extends React.Component {
   }
 }
 
+class CPExpressInstallInstructions extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    // TODO: layout with images
+    // TODO: get screenshot for image copy
+    // TODO: reset button gif- put in our cdn?
+    // TODO: make i18n strings for all the strings
+    return (
+      <div id="CPExpressInstallInstructions">
+        <h2>Install the Firmata firmware onto the CP Express</h2>
+        <ol>
+          <li>
+            Set your Circuit Playground Express to Bootloader Mode
+            <div style={style.oneColumn}>
+              <ul>
+                <li>
+                  Press or double-press the RESET button in the center of the
+                  board.
+                </li>
+                <li>
+                  You've successfully entered bootloader mode when the LEDs turn
+                  green and your computer detects a new removable storage device
+                  called 'CPLAYBOOT'.
+                </li>
+                <li>
+                  If pressing RESET once doesn't work, try double-pressing it.
+                </li>
+                <li>
+                  If the color LEDs turn all red, check your USB cable, try
+                  another cable or another USB port.
+                </li>
+              </ul>
+              <img
+                src="https://cdn-learn.adafruit.com/assets/assets/000/051/201/original/makecode_uf2.gif?1519151701"
```

Comment:
On my screen, it is difficult to see the change in the LED colors. Maybe we should create our own?

Reply:
Yes, I agree! I can't really see the color.

Reply:
Okay - cool. I'll create a video/gif and screenshot of the CPX in bootloader mode.

Reply:
Thank you for creating that! Resolving since ithis is updated in the new pr.